### ISSUE TITLE
Keyboard action search not working (neither enter key)

### ISSUE THREAD

**Issue by rolgalan** [arimorty/floatingsearchview#159]
Hi,

I'm starting to use your great library (which looks so cool!) but I've found a problem for getting **_onSearchAction()_** called.

While I'm writting text I succesfully get feedback from the **_OnQueryChangeListener()_**, but when I click the enter key, it only adds an space on the search text. For the action key, instead of having the magnifying glass icon I have the common enter icon.

I'm testing on a Nexus 5X, Android 7.0 (API 24).

I found the way to get the search edit text in an old issue (https://github.com/arimorty/floatingsearchview/issues/131) and I've fixed this issue on my code doing this, and now it works like a charm :) :
`        ((TextView) searchView.findViewById(R.id.search_bar_text)).setImeOptions(EditorInfo.IME_ACTION_SEARCH);`

I wanted to share this solution in case someone else faces this problem until the library is fixed.

Inspecting over the code, I think this might be solved by adding this line to [SearchInputView](https://github.com/arimorty/floatingsearchview/blob/ef70366635e19ba1c1337c7df7b2941f8856f756/library/src/main/res/layout/search_query_section.xml). (I'm not sure and have to time right now to test and push it, next week I can take a deeper loook):
`android:imeOptions="actionSearch" `

There was a similar issue https://github.com/arimorty/floatingsearchview/issues/57 some months ago, but apparently fixed on release 2.0.1.

Regards!

**Comment by strajky5** [comment]
i think, it has been fixed already :)

**Comment by rolgalan** [comment]
I've just discovered what was wrong!! 

**There is no problem on the library at all**. It was failing simply because I was setting _app:floatingSearch_showSearchKey="false"_ as the readme.md says. This attribute makes the whole _FloatingSearchView_ become into an useless _FloatingEditView_.

Today I launched the sample app and, after noticing it was working fine, I realized this value was setted as true there. It makes me feel so dumb now, but it's the value in the readme for the library, and never changed it.

Since there is no documentation about the custom xml attrs, maybe it should be changed in the example code on the readme.md.
